autotx/agents/ResearchTokensAgent.py

```diff
@@ -188,6 +193,24 @@ def _run(
                 if token["id"] in filtered_tokens_map
             ]
 
+            # If token is not in our token registry, we add it
+            current_network = COINGECKO_NETWORKS_TO_SUPPORTED_NETWORKS_MAP.get(
+                self.autotx.network.network
```

Comment:
network.network is weird

Reply:
agreed - should we do `self.autotx.network_info.network` ? or what would u suggest

Reply:
orrr maybe we could do `self.autotx.network.chain_instance` ? :thinking:

Reply:
I think `autotx.network.chain_id` since it's an `Enum` : https://github.com/polywrap/AutoTx/blob/c4518e4ed403f01339a88b15c6876b9a32a06eda/autotx/utils/ethereum/networks.py#L45

Reply:
Could also rename the import from `gnosis.eth` to `ChainId` so it's not as ambiguous.

Reply:
gotcha!

Reply:
i think it's fine keeping the same name from the import, only the name of the attribute in `AutoTx` is good enough

Reply:
ok i actually went ahead and changed the type from `EthereumNetwork` to `ChainId` - you were right @dOrgJelli lmk your thoughts